starfyre/compiler.py

```diff
@@ -56,10 +74,28 @@ def remove_empty_lines_from_end(lines):
 
     # regex pattern to match if a line is a css import, e.g. import "style.css"
     css_import_pattern = re.compile(r"^import\s[\"\'](.*?\.css)[\"\']")
-    
+
     with open(fyre_file_name, "r") as fyre_file:
         for line in fyre_file.readlines():
             css_import_match = css_import_pattern.search(line)
+
+            # check for fyre import styles
+            has_fyre_import = check_import_line(line=line, project_dir=project_dir)
+
+            # If the line is a fyre import statement, modify it to ensure proper resolution
+            if has_fyre_import:
+                # Split the line into module part and imported component
+                module_part, imported_component = line.split(" import ")
+
+                # Extract the file name to import from the module part
+                file_to_import = module_part.split(".")[-1]
+
+                # Get the name of the project directory
+                project_dir_name = str(project_dir).split("/")[-1]
+
+                # Modify the line to use the resolved import path
+                line = f'from ...build.{file_to_import} import {imported_component}'
```

Comment:
Hello @sansyrox, I have tried to do it the way you suggested whereby if the user imported as `test_application.parent` in the IR file, we should have `test_application.parent` in this IR file and let it somehow be resolved as `test_application.build.parent` in between. I tried this method but it proved difficult. I always ended up with Python telling me that it couldn't import `test_application.parent` which makes sense.

So later, I decided to stick with resolving it as a relative import:
> or if that doesn't work, we should use `build.{file_to_import}`

That's why this is the latest change. It replaces the `{project_dir_name}.build` with `...build`

Reply:
Hey @DeleMike , I have a question.